### DISCUSSION TITLE
更新后，chatgpt-mirai-qq-bot无法通过邮箱密码登录

### DISCUSSION THREAD

**Discussion by Nothingness-Void** [linweiyuan/go-chatgpt-api#210] [Q&A] [answered]
我之前的版本是用 xxxx/chatgpt/的版本，今天pull了一下更新成了/chatgpt/backend-api/的版本后，使用账号密码登录的全都失效了

**Comment by linweiyuan** [comment]
登录走的链接是 `/chatgpt/backend-api/login` ？

**Reply by Nothingness-Void** [reply]
大佬有什么办法解决吗。。。

**Reply by linweiyuan** [reply]
既然这样，可能需要加多一个接口映射下

**Reply by Nothingness-Void** [reply]
麻烦大佬了

**Reply by linweiyuan** [reply, marked as answer]
应该可以了，没试过，但是应该可以了

**Reply by Nothingness-Void** [reply]
大佬辛苦了，明天起床我去测试一下

**Comment by Nothingness-Void** [comment]
> 登录走的链接是 `/chatgpt/backend-api/login` ？

看报错是的

**Comment by cyclone12315** [comment]
哥你怎么解决的啊，我也是邮箱+密码登不了
![无标题](https://github.com/linweiyuan/go-chatgpt-api/assets/97504291/b08b751f-8cf9-42fb-b8b9-061f70e9dc1f)

**Reply by Nothingness-Void** [reply]
你搭建接入点的机器重新docker-compose pull一遍 最新版本就能用了 昨天半夜2点更新的

**Reply by cyclone12315** [reply]
pull过还是不行啊

**Reply by cyclone12315** [reply]
换成session token 就可以用，不晓得为啥

**Reply by linweiyuan** [reply]
不是 arm 机器吧，arm 镜像已不更新

**Reply by cyclone12315** [reply]
就是amd64的机器啊

![屏幕截图 2023-07-08 181715](https://github.com/linweiyuan/go-chatgpt-api/assets/97504291/ed2c17e8-4154-44f6-85b3-54e13232e7ae)

**Reply by linweiyuan** [reply]
原来你用的是我线上的服务，那就是最新的，400 就是账号密码错了

**Comment by cyclone12315** [comment]
emmm,邮箱输错了，现在可以了，，，那我自己搭在服务器的api该咋接入呢，那个browser_endpoint该填啥啊

**Reply by linweiyuan** [reply]
不变，IP 端口换成自己的即可

**Reply by Mu-Chen2** [reply]
就是你搭建完成之后 把配置文件的地址改成你搭建服务器的ip:端口 默认应该是8080 直接访问是Service go-chatgpt-api is ready.就可以用了

**Comment by cyclone12315** [comment]
邮箱登录报错变成500了，，，用IP+端口能直接访问的
![屏幕截图 2023-07-08 200806](https://github.com/linweiyuan/go-chatgpt-api/assets/97504291/3b0122f8-b224-4899-b184-c044d64c5e42)

**Reply by linweiyuan** [reply]
看上去这个不是 go-chatgpt-api 返回的错，看下日志

**Reply by cyclone12315** [reply]
我搞错了哈哈，后面忘了加chatgpt/backend-api/，加上就可以用了
